elife/python.sls

```diff
@@ -66,14 +66,7 @@ global-python-requisites:
 
 # DEPRECATED. removed after switch to 16.04
 
```

Comment:
does the 16.04 part need any change given the shift of the installation into builder?

Reply:
probably, I didn't look at it.

Reply:
The 16.04 all seems backward compatible, but cross-check please